cfg/aws/credentials.go

```diff
@@ -88,6 +88,12 @@ func getSession(config *aws.Config) *session.Session {
 		}
 	}
 	log.Printf("D! Successfully created credential sessions\n")
+	cred, err := ses.Config.Credentials.Get()
```

Comment:
Maybe I'm wrong but based on the reason below(Access Key) and also 
```
// Get returns the credentials value, or error if the credentials Value failed
// to be retrieved.
// Will return the cached credentials Value if it has not expired. If the
// credentials Value has expired the Provider's Retrieve() will be called
// to refresh the credentials.
//
// If Credentials.Expire() was called the credentials Value will be force
// expired, and the next call to Get() will cause them to be refreshed.
```

`Get` will be forced to [refresh the credentials ](https://github.com/aws/aws-sdk-go/blob/main/aws/credentials/credentials.go#L254-L256)but is it within the scope of this PR and also CWAgent also has a [mechanism to refresh the credentials](https://github.com/aws/amazon-cloudwatch-agent/blob/d9516317422e22618e56eeb9b70ec8487e989765/cfg/aws/credentials.go#L228-L245). So will it change the behavior of these credentials?
Therefore, would suggest change to use 
```
ses.Config.Credentials.Retrieve() https://github.com/aws/aws-sdk-go/blob/main/aws/credentials/credentials.go#L137-L139
```

Reply:
I don't follow. Why would I use `Retrieve()`? We want to use the cached value if it isn't expired.

Reply:
My bad. Looking at the code again, the `Get` is using `Retrieve` under the hood since originally I thought the `Get` will refresh the credentials if it is expired and also `Get` exposes the AccessKey and SecretKey while `Retrieve` only expose CredentialProvider.